backend/src/schema/resolvers/users/location.spec.js

```diff
@@ -85,6 +95,62 @@ beforeEach(() => {
 
 afterEach(cleanDatabase)
 
+describe('Location Service',() => {
+  // Authentication
+  // TODO: unify, externalize, simplify, wtf?
+  let user
+  beforeEach(async () => {
```

Comment:
Why beforeEach? We can use the same user for all following it-blocks. So I would suggest beforeAll to create the user. Then may be beforeEach to set the authenticated user (it is set to null beforeEach in the head), or set the authenticated user in each async it-block explecitely